client/src/main/scala/org/http4s/client/DefaultClient.scala

```diff
@@ -39,18 +39,18 @@ private[http4s] abstract class DefaultClient[F[_]](implicit F: Bracket[F, Throwa
     * @return The result of applying f to the response to req
     */
   def fetch[A](req: F[Request[F]])(f: Response[F] => F[A]): F[A] =
-    req.flatMap(fetch(_)(f))
+    req.flatMap(run(_).use(f))
```

Comment:
In a separate PR, it makes sense to deprecate this one.  It's the same hidden use.